test_paths_between.py

```diff
@@ -0,0 +1,23 @@
+from main import init_system
+# from pathselection import joinpathselection
+#from integrating_civilizer import *
+from api.apiutils import Relation
+api, reporting = init_system("../testmodel/")
+
+
+t1 = "Employee_directory.csv"
```

Comment:
since these are dataset specific, can you move them to inside the main() instead (bottom of the file?)